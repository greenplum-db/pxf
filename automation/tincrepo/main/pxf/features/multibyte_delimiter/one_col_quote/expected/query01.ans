-- start_ignore
-- end_ignore
-- @description query01 for PXF Multibyte delimiter, one col with quote
SELECT * from pxf_multibyte_onecol_quote_data ORDER BY s1;
                    s1
-------------------------------------------
  minden  amire  szksge  van  a  szeretet
  rgles  d'automation
  t  s  seo  le  tstil  dea-
(3 rows)

