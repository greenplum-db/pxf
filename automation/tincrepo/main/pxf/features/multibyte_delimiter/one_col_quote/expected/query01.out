-- @description query01 for PXF Multibyte delimiter, one col with quote
SELECT * from pxf_multibyte_onecol_quote_data ORDER BY s1;
                  s1                  
--------------------------------------
 minden amire szüksége van a szeretet
 règles d'automation
 tá sé seo le tástáil dea-
(3 rows)

